### PR TITLE
[package_info] Add package_info_tizen package

### PR DESCRIPTION
**Pull request by bwikbs**

### CODE REVIEW

File: packages/package_info/example/integration_test/package_info_test.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    expect(info.packageName, 'io.flutter.plugins.packageinfoexample');
+    expect(info.version, '1.0');
+    expect(
+      log,
+      <Matcher>[
+        isMethodCall('getAll', arguments: null),
+      ],
+    );
+  });
+}
```
Comment (swift-kim) [inline packages/package_info/example/integration_test/package_info_test.dart:49]: You may replace this file with [this](https://github.com/swift-kim/plugins/commit/c07b89dd545bd29fa016e512898c7acc01b9fb16) (a new derivative of [package_info_test.dart](https://github.com/flutter/plugins/blob/master/packages/package_info/example/integration_test/package_info_test.dart)) which better evaluates the functionality of this plugin on Tizen devices. You will also have to update `example/lib/main.dart` as described in the commit to make the tests pass on devices with tiny screens (wearables).

Comment (bwikbs) [reply packages/package_info/example/integration_test/package_info_test.dart:29]: done.

File: packages/package_info/tizen/src/package_info_tizen_plugin.cc [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    char *version = NULL;
+    char *label = NULL;
+    package_info_h package_info = NULL;
+    flutter::EncodableMap msg;
+
+    int ret = app_get_id(&app_id);
+    if (ret != APP_ERROR_NONE || app_id == NULL) {
+      result->Error(
+          std::to_string(ret), "Failed to get app_id",
+          flutter::EncodableValue(appErrorToString(ret)));  // LCOV_EXCL_LINE
```
[the remark this answers is not in the document]
Comment (bwikbs) [reply packages/package_info/tizen/src/package_info_tizen_plugin.cc]: done.

File: packages/package_info/lib/package_info_tizen.dart [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// You have generated a new plugin project without
```
[the remark this answers is not in the document]
Comment (bwikbs) [reply packages/package_info/lib/package_info_tizen.dart]: done.

**Review by seungsoo47 — APPROVED** [review]

**Pull request merged**